imports/plugins/included/payments-stripe/server/no-meteor/util/stripeCreateRefund.js

```diff
@@ -10,16 +10,17 @@ import formatForStripe from "./formatForStripe";
  * @param {Object} context an object containing the per-request state
  * @param {Object} paymentMethod object containing transaction ID
  * @param {Number} amount the amount to be refunded
+ * @param {Number} [reason] the reason for the refund
```

Comment:
(Repeat) Can we link to the stripe docs describing the allowed values for `reason` here or put in a cheat sheet?

Reply:
Yes... I also have an explanation in the comments in the client side code ( #5158 )